src/world/action/land.c

```diff
@@ -1,9 +1,206 @@
 #include "common.h"
+#include "functions.h"
 
-INCLUDE_ASM(void, "world/action/land", func_802B6000_E24920, void);
+void func_802B62CC_E24BEC(void);
+void func_802B644C_E24D6C(void);
 
-INCLUDE_ASM(void, "world/action/land", func_802B61C0_E24AE0, void);
+void func_802B6000_E24920(void) {
+    f32 inputMoveMagnitude;
+    f32 inputMoveAngle;
+    s32 jumpInputCheck;
+    s32 phi_a0;
+    CollisionStatus* currentCollisionStatus = &gCollisionStatus;
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    Camera* currentCameras = &gCameras[0];
+    
+    if ((playerStatus->animFlags & 0x1000) != 0) {
+        func_802B62CC_E24BEC();
+        return;
+    }
 
-INCLUDE_ASM(void, "world/action/land", func_802B62CC_E24BEC, void);
+    if (playerStatus->flags < 0) {
+        playerStatus->flags = playerStatus->flags & 0x7F77FFF1;
```

Comment:
Similar to the 0xFFFBFFFF, this is a bunch of flags ORd together and then inverted

Reply:
I'm currently trying to use PlayerStatusFlags for this, but it seems that some flags are missing.  Would I just add them to the enum? (Specifically 0x80000 and 0x8)